orchestrator-salt/src/main/resources/salt-common/salt/fluent/template/databus_metering.conf.j2

```diff
@@ -23,14 +23,11 @@
     event_message_field              message
     headers                          {{ fluent.dbusMeteringAppHeaders }}
     stream_name                      {{ fluent.dbusMeteringStreamName }}
-    partition_key                    "#{Socket.gethostname}"
-    {%- if fluent.proxyUrl %}
-    proxy_url                        "{{ fluent.proxyUrl }}"
-    {%- if fluent.proxyAuth %}
+    partition_key                    "#{Socket.gethostname}"{%- if fluent.proxyUrl %}
+    proxy_url                        "{{ fluent.proxyUrl }}"{%- if fluent.proxyAuth %}
     proxy_username                   "{{ fluent.proxyUser }}"
-    proxy_password                   "{{ fluent.proxyPassword }}"
-    {% endif %}
-    {% endif %}
+    proxy_password                   "{{ fluent.proxyPassword }}"{% endif %}{% endif %}{% if fluent.noProxyHosts and fluent.fluentVersion > 3 %}
+    no_proxy                         "{{ fluent.noProxyHosts }}"{% endif %}
```

Comment:
should be inside the {%- if fluent.proxyUrl %} but after {%- if fluent.proxyAuth %} 's endif

Reply:
could be, note that option is not used if proxy url is not filled, so won't have any affect if it's there, also the no proxy fluent variable will be also None if proxy options are not filled, so the option would not end up there anyway, i can change if you think but won't really have any impact